pallets/staking/src/lib.rs

```diff
@@ -98,7 +98,7 @@
 //! An account can become a validator candidate via the
 //! [`validate`](./enum.Call.html#variant.validate) call
 //! But only those validators are in effect whose compliance status is active via
-//! [`add_permissioned_validator`](./enum.Call.html#variant.validate) call & there _stash_ accounts has valid CDD claim.
+//! [`add_permissioned_validator_entity`](./enum.Call.html#variant.validate) call & there _stash_ accounts has valid CDD claim.
```

Comment:
nit - can we keep the old name - entity is really the PUIS ID granularity.

Reply:
We are not really permission for the validator directly. But if you think it is better to keep the same name then I can do it.